tests/test_asgi_server.py

```diff
@@ -0,0 +1,6 @@
+import autover
+from starlette.applications import Starlette
+
+print(autover.__version__)
+
+app = Starlette()
```

Comment:
Is this file a "test support file" rather than an actual test? If so, maybe rename it so it's clearly not a file containing tests, and then no need to exclude specially from nose? (Unless there's some reason that's not possible?)

Reply:
I did it that way just to make sure that the file got bundled when creating the package (as everything that is named `test*` inside the tests folder does), but I can just modify `setup.py` to force the file inclusion there after renaming. I don't know which one is more elegant...

Reply:
> as everything that is named test* inside the tests folder does

Do you happen to know if that is a limitation we have put in somewhere here in autover? I wasn't aware of this as a general thing (in setup tools, tox, etc), and I'm only on my phone just now...